desktop.html

```diff
@@ -0,0 +1,247 @@
+            <ul class="skills">
+                <li class="sbox">HTML</li>
+                <li class="sbox">CSS</li>
+                <li class="sbox">JavaScript</li>
+            </ul>
+
+            <input class="btn" type="button" value="See project">
+        </div>
+        </div>
+    </section>
+
+    <section class="about">
+        <div class="about1">
+        <h2>About Myself</h2>
+        <p>
+            Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        
+        <h3 class="connect2">LET'S CONNECT</h3>
+        
+        <ul class="icons">
+            <li>
+                <a href="https://twitter.com"><img src="./assets/images/twitter.png"></a>
+            </li>
+
+            <li>
+                <a href="https://www.linkedin.com/in/fatima-najafi-75424a23a/"><img src="./assets/images/linkedin.png"></a>
+            </li>
+
+            <li>
+                <a href="#"><img src="./assets/images/M.png"></a>
+            </li>
+
+            <li>
+                <a href="https://github.com/fatima-najafi"><img src="./assets/images/github.png"></a>
+            </li>
+
+            <li>
+                <a href="#"><img src="./assets/images/victory.png"></a>
+            </li>
+        </ul>
+
+        <input class="btn" id="resume" type="button" value="Get my resume">
+        </div>
+
+        <ul class="about2">
+            <li class="tskill">
+              <p>Languages</p>
+              <a href="#"><img src="./assets/images/down.png" alt="drop"></a>
+            </li>
+
+            <li>
+              <ul class="lang">
+                <li>
+                  <img src="./assets/images/js.png" alt="javascript" />
+                  <p>JavaScript</p>
+                </li>
+
+                <li>
+                  <img src="./assets/images/html.png" alt="html" />
+                  <p>HTML</p>
+                </li>
+                <li>
+                  <img src="./assets/images/css.png" alt="css" />
+                  <p>CSS</p>
+                </li>
+              </ul>
+            </li>
+            <li class="tskill">
+              <p>Frameworks</p>
+              <a href="#"><img src="./assets/images/right.png" alt="right"></a>
+            </li>
+            <hr />
+            <li class="tskill">
+              <p>Skills</p>
+              <a href="#"><img src="./assets/images/right.png" alt="right"></a>
+            </li>
+            <hr />
+          </ul>
+    </section>
+    <section class="contact-section">
+        <section class="contact">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+                I’d love to help with it!</p>
```

Comment:
- According to [this requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=You%20must%20replicate%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20info%20from%20the%20Figma%20templates.), Kindly make sure to stick to the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible.

**Figma Design**
![Screenshot from 2023-05-06 11-43-34](https://user-images.githubusercontent.com/97834160/236617446-ab35a7c8-2499-44f2-b8c4-6fc0abe1952b.png)

**Your Design**
![Screenshot from 2023-05-06 11-44-40](https://user-images.githubusercontent.com/97834160/236617454-e5b263ea-7470-4cc4-aa52-719c3059f90f.png)

- Kindly make sure to use the Background image provided by the [FIgma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- I think it would be better to use the font-family provided by [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
- Kindly make sure to respect the sizes for the placeholders, as well as the margins to match the template provided by [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) for the desktop version of your portfolio